dingo/core/utils/pt_to_hdf5.py

```diff
@@ -70,6 +69,9 @@ def main():
         f.attrs['CANONICAL_FILE_BASENAME'] = os.path.basename(out_file_name)
         # Add a few metadata entries as attributes
         f.attrs['approximant'] = d['metadata']['dataset_settings']['waveform_generator']['approximant']
+        f.attrs['epoch'] = d['epoch']
+        # Add the dingo version used for training
+        f.attrs['dingo_version'] = str(d.get('version'))
```

Comment:
Can this key be called simply "version"? That would keep it consistent with other Dingo files.
```suggestion
        f.attrs['version'] = str(d.get('version'))
```